widget/embedded/src/components/SwapDetails/SwapDetails.tsx

```diff
@@ -188,6 +188,14 @@ export function SwapDetails(props: SwapDetailsProps) {
     USD_VALUE_MAX_DECIMALS
   );
 
+  const realOutputUsdValue = outputAmount
+    ? String(Number(outputAmount) * (lastStep.toUsdPrice || 0))
+    : '';
+
+  const realInputUsdValue = String(
+    parseFloat(swap.inputAmount) * (firstStep.fromUsdPrice || 0)
+  );
+
```

Comment:
Using BigNumber.js is recommended for precision and consistency.

Reply:
fixed.